package.json

```
@@ -5,68 +5,77 @@
     "author": "shiy2008@gmail.com",
     "dependencies": {
         "@editorjs/code": "^2.9.0",
-        "@editorjs/editorjs": "^2.29.1",
-        "@editorjs/header": "^2.8.1",
-        "@editorjs/image": "^2.9.0",
+        "@editorjs/editorjs": "^2.30.5",
+        "@editorjs/header": "^2.8.7",
+        "@editorjs/image": "^2.9.3",
         "@editorjs/link": "^2.6.2",
-        "@editorjs/list": "^1.9.0",
-        "@editorjs/paragraph": "^2.11.5",
-        "@editorjs/quote": "^2.6.0",
+        "@editorjs/list": "^1.10.0",
+        "@editorjs/paragraph": "^2.11.6",
+        "@editorjs/quote": "^2.7.2",
         "browser-fs-access": "^0.35.0",
         "browser-unhandled-rejection": "^1.0.2",
-        "echarts-jsx": "^1.2.0",
+        "echarts-jsx": "^1.2.1",
         "editorjs-html": "^3.4.3",
         "idea-react": "^2.0.0-rc.2",
-        "koajax": "^1.1.2",
+        "koajax": "^3.0.2",
         "lodash": "^4.17.21",
-        "mobx": "^6.12.4",
-        "mobx-github": "^0.3.2",
+        "mobx": "^6.13.2",
+        "mobx-github": "^0.3.4",
         "mobx-i18n": "^0.5.0",
         "mobx-react": "^9.1.1",
-        "mobx-restful": "^1.0.0",
-        "mobx-restful-table": "^2.0.0-rc.1",
-        "preact": "^10.22.0",
-        "react-bootstrap": "^2.10.3",
+        "mobx-restful": "^1.0.1",
+        "mobx-restful-table": "^2.0.0",
+        "react": "18.3.1",
+        "react-dom": "18.3.1",
```

```suggestion
        "react": "^18.3.1",
        "react-dom": "^18.3.1",
```
前端库一般不需要锁版本，你需要重新执行一下安装命令更新锁文件。